src/track/heaptrack.sh.cmake

```diff
@@ -117,7 +117,15 @@ while true; do
                 echo "Missing PID argument."
                 exit 1
             fi
-            client=$(cat /proc/$pid/comm)
+            case $(uname) in
```

Comment:
I think we can always use cmdline, that also exists on linux. but it would include the arguments too, which we probably don't want to output on freebsd either. so something like this is hopefully simpler and portable?
```
client=$(awk -F'\0' '{print $1}' < /proc/$pid/cmdline)
```

Reply:
oh, comm doesn't include arguments?

> `awk -F'\0' '{print $1}'`

Weird: this works with `gawk` (GNU awk) but with FreeBSD awk it only returns the first character. But `awk '{print $1}'` without the custom separator returns the first argument just fine..

Reply:
hehe but that doesn't work on linux ;-)

in that case, I'm fine with leaving it as is